test/functional/tests/io_class/test_io_class_directory.py

```diff
@@ -178,51 +199,67 @@ def test_ioclass_directory_file_operations(filesystem):
 
     with TestRun.step("Create test file."):
         classified_before = cache.get_io_class_statistics(
-            io_class_id=ioclass_id).usage_stats.occupancy
+            io_class_id=ioclass_id
+        ).usage_stats.occupancy
         file_path = f"{test_dir_path}/test_file"
-        (Dd().input("/dev/urandom").output(file_path).oflag("sync")
-         .block_size(Size(1, Unit.MebiByte)).count(dd_blocks).run())
+        (
+            Dd()
+            .input("/dev/urandom")
+            .output(file_path)
+            .oflag("sync")
+            .block_size(Size(1, Unit.MebiByte))
+            .count(dd_blocks)
+            .run()
+        )
         time.sleep(ioclass_config.MAX_CLASSIFICATION_DELAY.seconds)
         sync()
         drop_caches(DropCachesMode.ALL)
         test_file = File(file_path).refresh_item()
 
     with TestRun.step("Check classified occupancy."):
         classified_after = cache.get_io_class_statistics(
-            io_class_id=ioclass_id).usage_stats.occupancy
+            io_class_id=ioclass_id
+        ).usage_stats.occupancy
         check_occupancy(classified_before + test_file.size, classified_after)
 
     with TestRun.step("Move test file out of classified directory."):
         classified_before = classified_after
-        non_classified_before = cache.get_io_class_statistics(io_class_id=0).usage_stats.occupancy
+        non_classified_before = cache.get_io_class_statistics(
+            io_class_id=0
+        ).usage_stats.occupancy
         test_file.move(destination=mountpoint)
         time.sleep(ioclass_config.MAX_CLASSIFICATION_DELAY.seconds)
         sync()
         drop_caches(DropCachesMode.ALL)
 
     with TestRun.step("Check classified occupancy."):
         classified_after = cache.get_io_class_statistics(
-            io_class_id=ioclass_id).usage_stats.occupancy
+            io_class_id=ioclass_id
+        ).usage_stats.occupancy
         check_occupancy(classified_before, classified_after)
         TestRun.LOGGER.info("Checking non-classified occupancy")
-        non_classified_after = cache.get_io_class_statistics(io_class_id=0).usage_stats.occupancy
+        non_classified_after = cache.get_io_class_statistics(
+            io_class_id=0
+        ).usage_stats.occupancy
         check_occupancy(non_classified_before, non_classified_after)
 
     with TestRun.step("Read test file."):
         classified_before = classified_after
         non_classified_before = non_classified_after
-        (Dd().input(test_file.full_path).output("/dev/null")
-         .iflag("sync").run())
+        (Dd().input(test_file.full_path).output("/dev/null").iflag("sync").run())
```

Comment:
No need for the outermost parenthesis, unless you'll format it like all the other Dd().